packages/gestalt/src/Icon.js

```diff
@@ -34,6 +34,20 @@ type Props = {|
 
 const IconNames = Object.keys(icons);
 
+const noFlipOnRtlIconNames = [
+  'facebook',
```

Comment:
@selena-zeng this is different from the approach we talked about - is this in agreement with PDS, that only these icons should not be flipped and all the other ones should?